projects/kit/components/range/range.component.ts

```diff
@@ -104,17 +104,37 @@ export class TuiRangeComponent
     }
 
     get fractionStep(): number {
-        return this.step / (this.max - this.min);
+        return this.step / ((this.max ?? 100) - (this.min ?? 0));
```

Comment:
```suggestion
        return this.step / (this.computedMax - this.computedMin);
```

Reply:
no need due to https://github.com/Tinkoff/taiga-ui/pull/4848#discussion_r1258190049